DTDL/valuechange_options.json

```diff
@@ -0,0 +1,33 @@
+{
+    "@context" : "dtmi:dtdl:context;2",
+    "@id" : "dtmi:io:tributech:options:tcads;1", 
+    "@type": "Interface", 
+    "extends": "dtmi:io:tributech:options:base;1",
+    "displayName": "Beckhoff TwinCat ADS Value Options", 
+    "contents" : [
+        {
+            "@type" : "Property", 
+            "name" : "PMIN", 
+            "schema" : "integer", 
+            "writable" : true,
+            "displayName" : "Minimum Period (Seconds)", 
+            "description" : "Minimum Time Difference between Value Changes (Value will be ignored if it changes faster)"
+        }, 
+        {
+            "@type" : "Property", 
+            "name" : "PMAX", 
+            "schema" : "integer", 
+            "writable" : true,
+            "displayName" : "Maximum Period (Minutes)", 
```

Comment:
@Badabunga really minutes? I would highly prefer to use the same unit for PMIN and PMAX. Could we use seconds on both ends?

Reply:
I just thought of being a User : 

- How often do I use Time Periods like Seconds in a Maximum Time

And my perception was that I will only use this for Variables which I suspect will not change very often. So do I need Seconds for a Value which will rarely change ? 
That's why I choose Minutes instead of Seconds because I thought it was more useful for the User than Seconds.  

I see the Advantage of using the same Time Unit. But I think it will be easier for the User to use Minutes.

Reply:
A user will IMO read the comment for PMIN and set PMIN to 10 and PMAX to 3600 (1h). And this configuration would be wrong. Please decide. It's correct in both versions, it's my opinion towards usabilty.

Reply:
Changed it to Seconds